### ISSUE TITLE
How do you silence the email notifications?

### ISSUE THREAD

**Issue by paulcr** [githubschool/foundations-4-28-15#124]

**Comment by patrickmckenna** [comment]
@paulcr Great question. Go to your settings page, the gear icon in the upper right corner. Then click on the notifications tab.

**Comment by paulcr** [comment]
@PatrickBM I have notifications I have to watch in a couple of other repos and this class is flooding my inbox. Can I just silence the notifications from this repo?

**Comment by paulcr** [comment]
@PatrickBM I know you're probably getting bombarded with questions.... I've unwatched this repo but can you guys at least stop mentioning the entire organization in comments to quiet the emails down a bit? It's very hard to follow along.

**Comment by patrickmckenna** [comment]
@paulcr Sorry for the delayed response! Glad you figured out how to unwatch the repo. Afraid you'll have to just delete all of those. You could also turn off email notifications temporarily, or even [remove yourself](https://help.github.com/articles/removing-yourself-from-an-organization/) from the org.
